### ISSUE TITLE
4x missing alternative text https://uclalibrary.github.io/research-tips/sandbox/resource-playlist-study-tips/

### ISSUE THREAD

**Issue by woof84** [UCLALibrary/research-tips#199]

**Comment by woof84** [comment]
fixed and updated metadata
